### PR TITLE
Update BAT tests

### LINKED ISSUE #7: Update "ledger" section after BAT is live

### PR DESCRIPTION
**Pull request by srirambv**
Closes #7

### CODE REVIEW

**Comment by LaurenWags** [comment]
++ looks good to me, will wait for @kjozwiak to give input.

File: wikitemplate-cr-upgrade [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+## Performance test
+
+_Each start should take less than 7 seconds_
+- [ ] Enable only sync (new sync group).
+- [ ] Enable only sync with a large sync group (many entries).
+- [ ] Enable only payments.
+- [ ] Only import a large set of bookmarks.
+- [ ] Combine sync, payments, and a large set of bookmarks.
+
+## Ledger
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: add space below `## Ledger` to keep document consistent

File: wikitemplate-cr-upgrade [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+- [ ] Verify wallet is auto created after enabling payments
+- [ ] Verify monthly budget and account balance shows correct BAT and USD value
+- [ ] Verify the wallet overlay is shown when wallet transition is happening upon upgrade 
+- [ ] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
+- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
+- [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table.
+- [ ] Check that disabling payments and enabling them again does not lose state.
+
+
+## Sync
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: add space below `## Sync` to keep document consistent

File: wikitemplate-cr-upgrade [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+## Bookmarks
+
+- [ ] Test that creating a bookmark on the bookmarks toolbar works
+- [ ] Test that creating a bookmark folder on the bookmarks toolbar works
+- [ ] Test that moving a bookmark into a folder by drag and drop on the bookmarks folder works
+- [ ] Test that clicking a bookmark in the toolbar loads the bookmark.
+- [ ] Test that clicking a bookmark in a bookmark toolbar folder loads the bookmark.
+
+## Context menus
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: add space below `## Context menus` to keep document consistent

File: wikitemplate-cr-upgrade [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+- [ ] Go to https://trac.torproject.org/projects/tor/login and make sure that the password can be saved. Make sure the saved password shows up in `about:passwords`. Then reload https://trac.torproject.org/projects/tor/login and make sure the password is autofilled.
+- [ ] Open a github issue and type some misspellings, make sure they are underlined.
+- [ ] Make sure that right clicking on a word with suggestions gives a suggestion and that clicking on the suggestion replaces the text.
+- [ ] Make sure that Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab but does NOT switch to it.  Click on it and make sure it is already loaded.
+- [ ] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works.
+- [ ] Test that PDF is loaded at http://www.orimi.com/pdf-test.pdf
+- [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
+
+## Flash tests
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: add space below `## Flash tests` to keep document consistent

File: wikitemplate-cr-upgrade [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+- [ ] Test that flash placeholder appears on http://www.homestarrunner.com
+- [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com
+
+## Autofill tests
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: add space below `## Autofill tests` to keep document consistent

File: wikitemplate-cr-upgrade [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+## Ledger
+- [ ] Verify wallet is auto created after enabling payments
+- [ ] Verify monthly budget and account balance shows correct BAT and USD value
+- [ ] Verify the wallet overlay is shown when wallet transition is happening upon upgrade 
+- [ ] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
+- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
+- [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table.
+- [ ] Check that disabling payments and enabling them again does not lose state.
+
```
Comment (kjozwiak) [inline wikitemplate-cr-upgrade]: nitpick: remove extra space

**Comment by srirambv** [comment]
@kjozwiak  updated as per feedback

**Comment by kjozwiak** [comment]
@srirambv looks like the PR has some conflicts, mind rebasing? Let me know if you need any help with it.

**Pull request merged**